editor/components/NotesModal.tsx

```diff
@@ -31,26 +31,24 @@ export default function NotesModal({
     formState: { errors },
     reset,
   } = useForm<NoteFormInputs>();
-  const { mutate, loading: isSubmitting } = useMutation(INSERT_CRASH_NOTE);
+  const { mutate, loading: isSubmitting } = useMutation(insertMutation);
 
   const onSubmit: SubmitHandler<NoteFormInputs> = async (data) => {
     const noteData = {
       ...data,
-      crashPk: crashPk,
+      recordId: recordId,
       userEmail: user?.email,
     };
-    const responseData = await mutate<{
-      insert_crash_notes_one: { returning: CrashNote };
-    }>(noteData);
-    if (responseData && responseData.insert_crash_notes_one) {
-      onSubmitCallback(responseData.insert_crash_notes_one.returning);
+    const responseData = await mutate(noteData);
+    if (!!responseData) {
+      onSubmitCallback();
```

Comment:
i realized this callback doesnt actually need arguments, so i was able to generalize this part of the function